win32ss/gdi/ntgdi/line.c

```diff
@@ -153,6 +153,7 @@ IntGdiLineTo(DC  *dc,
     RECTL     Bounds;
     POINT     Points[2];
     PDC_ATTR pdcattr = dc->pdcattr;
+    PPATH     pPath;
     ASSERT_DC_PREPARED(dc);
```

Comment:
Something to think about 🤔
```suggestion
    PDC_ATTR pdcattr;
    PPATH     pPath;

    ASSERT_DC_PREPARED(dc);

    pdcattr = dc->pdcattr;
```

Reply:
OK.